pom.xml

```diff
@@ -66,16 +66,25 @@
     <checkstyle.config>${project.build.directory}/plugin-config/checkstyle/eg-checkstyle.xml</checkstyle.config>
     <mycila.config.header>${project.build.directory}/plugin-config/mycila/APACHE-2.txt</mycila.config.header>
     <mycila.config.header.def>${project.build.directory}/plugin-config/mycila/eg-definitions.xml</mycila.config.header.def>
+    
+    <!-- deployment properties -->
+    <nexus.staging.maven.plugin.repository.serverId>sonatype-nexus-staging</nexus.staging.maven.plugin.repository.serverId>
```

Comment:
The changes look ok, just curious as to why you need them? Also, could you please put the properties here in ascending alphabetical order to make it easier to read them?